src/main/java/mate/academy/dao/jdbc/ManufacturerDaoJdbc.java

```diff
@@ -0,0 +1,117 @@
+package mate.academy.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.dao.ManufacturerDao;
+import mate.academy.exception.DataProcessingException;
+import mate.academy.lib.Dao;
+import mate.academy.model.Manufacturer;
+import mate.academy.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoJdbc implements ManufacturerDao {
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String query = "INSERT INTO manufacturers (manufacturer_name, manufacturer_country)"
+                + " VALUES (?,?);";
+        String querySelectAll = "SELECT * FROM manufacturers;";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement statement = connection.prepareStatement(query)) {
+            statement.setString(1, manufacturer.getName());
+            statement.setString(2, manufacturer.getCountry());
+            statement.executeUpdate();
+            ResultSet resultSet = statement.executeQuery(querySelectAll);
+            while (resultSet.next()) {
+                Long id = resultSet.getObject("manufacturer_id", Long.class);
+                manufacturer.setId(id);
+            }
+            return manufacturer;
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't create manufacturer: " + manufacturer, e);
+        }
+    }
+
+    @Override
+    public Optional<Manufacturer> get(Long id) {
+        String query = "SELECT * FROM manufacturers WHERE manufacturer_id = ?;";
```

Comment:
```suggestion
        String getByIdQuery = "SELECT * FROM manufacturers WHERE manufacturer_id = ?;";
```
make your string a bit more informative